docs/application/native/guides/device/system.md

```diff
@@ -644,6 +645,17 @@ The following table lists the system setting feature keys:
 | `http://tizen.org/feature/systemsetting.lock_screen` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/systemsetting` key, if the device supports a way to change or get the lock screen wallpaper. |
 | `http://tizen.org/feature/systemsetting.notification_email` | `bool` | The platform returns `true` for this key and the `http://tizen.org/feature/systemsetting` key, if the device supports a way to change or get a ringtone for all email notifications. |
 
+The following table lists the thermistor feature keys:
+
+<a name="thermistor"></a>
+**Table: Thermistor feature keys**
+
+| Key                                      | Type   | Description                              |
+|------------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/thermistor.ap` | `bool` | The platform returns `true` for this key, if the device supports a thermistor for Application Processor. |
+| `http://tizen.org/feature/thermistor.cp` | `bool` | The platform returns `true` for this key, if the device supports a thermistor for Communications Processor. |
```

Comment:
Suggestion: To maintain consistency in the page, please rephrase to:

The platform returns `true` for this key, if the device supports a thermistor for the communications processor.

I have added "the" as processor here is a qualifier. Also, in the "Table: Platform system keys" section, we have used sentence case.